Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -710,3 +712,22 @@ void CEXIETHERNET::BuiltInBBAInterface::RecvStop()
   m_queue_write = 0;
 }
 }  // namespace ExpansionInterface
+
+BbaTcpSocket::BbaTcpSocket() = default;
+
+sf::Socket::Status BbaTcpSocket::Connect(const sf::IpAddress& dest, u16 port, u32 net_ip)
+{
+  sockaddr_in addr;
+  addr.sin_addr.s_addr = net_ip;
+  addr.sin_family = AF_INET;
+  addr.sin_port = 0;
```

Comment:
Should `sf::Socket::AnyPort` be used for consistency?

Reply:
Since I'm using the C socket interface `::bind`, I'd rather not to. There is no guarantee that `sf::Socket::AnyPort` won't be changed to something other than 0.